src/app/client/src/app/plugins/profile/components/profile-page-redesign/profile-page-redesign.component.css

```diff
@@ -0,0 +1,78 @@
+ 
+/* fonts */
+  .font-w-bold {
+    font-weight: bold !important;
+  }
+
+  /* colors */
+  .color-2 {
+    color: #007aff !important;
+  }
+  .color-5 {
+    color: #9017FF !important;
+  }
+
+  
+
+  .ui.small.custom.circular.image {
+    height: 75px;
+    width: 75px;
+    max-height: 75px;
+    max-width: 75px; 
+  }
+
+  .badges-container {
+    max-width: calc(195px * 4);
+  }
+
+
+  /* profile page color palate */
+
+  .profile-bg-c-1 {
+    background: #fff !important;
+  }
+  .profile-bg-c-2 {
+    background: #fafafa !important;
+  }
+  .profile-bg-c-3 {
+    background: #196DAB !important;
```

Comment:
@prajwal88 instead of using hardcoded color values.. Use scss variables. It will help in theming from single place.

Reply:
I will create the variables after ur code is merged. since u are adding configuration for it in angular.json right